ios/private/common_utils.bzl

```diff
@@ -16,49 +15,6 @@ def ios_bundle_module_shim(name):
         cmd = "sed 's/PLACEHOLDER/" + name + "/g' < $< > $@",
     )
 
-def assemble_pod(
-        name,
-        podspec = "",
-        srcs = [],
```

Comment:
Moved to a separate file with `assemble_ios_release`. Minor organizational change.